js/map.js

```diff
@@ -0,0 +1,58 @@
+import { activatePage } from './page-state.js';
+import { generateAd } from './data.js';
+import { renderOffer } from './card.js';
+
+
+const TOKYO = {
+  lat: 35.68950,
+  lng: 139.69171,
+};
```

Comment:
Давай нейминг вот так поменяем
```suggestion
const TokyoCoords = {
  LAT: 35.68950,
  LNG: 139.69171,
};
```